crates/dbsp/src/circuit/metadata.rs

```diff
@@ -27,6 +27,12 @@ pub const SHARED_BYTES_LABEL: &str = "shared bytes";
 /// operator, e.g., the number of entries in a trace.
 pub const NUM_ENTRIES_LABEL: &str = "total size";
 
+/// The number of input tuples ingested by the operator.
+pub const NUM_INPUTS: &str = "inputs";
```

Comment:
why not call it input_tuple_count?
I always have trouble understanding such generic names.

Reply:
the previous metric should be called exactly that "state_size_in_tuples"